pkg/netstore/netstore_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package netstore_test
+
+import (
+	"bytes"
+	"context"
+	"sync/atomic"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/netstore"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/storage/mock"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var chunkData = []byte("mockdata")
+
+type mockValidator struct{}
+
+func (m mockValidator) Validate(_ swarm.Chunk) bool { return true }
```

Comment:
An extremely minor suggestion, `m` receiver is not needed.

Reply:
but it has to implement the interface (the validator is an interface, not a function type alias)

Reply:
This is very very minor, but just as `swarm.Chunk` is `_`, `func (_ mockValidator) ... ` can have `_`.